__tests__/components/dashtable_v2.test.jsx

```diff
@@ -0,0 +1,161 @@
+import { render } from '@testing-library/react';
+import React from 'react';
+import GlobalDashboardTable from '../../components/dashtable_v2.js';
+
+describe('GlobalDashboardTable', () => {
```

Comment:
This will be updated in an amended PR - will also be removing the redundant Jest PR.